package-lock.json

```diff
@@ -51,7 +51,7 @@
                 "highlight.js": "*",
                 "html-webpack-plugin": "^4.5.1",
                 "jest": "^26.6.3",
-                "jest-playwright-preset": "^1.4.3",
+                "jest-playwright-preset": "^1.4.4",
```

Comment:
I pulled locally and updated the `package-lock.json`. I think your version got a lot of changes due to a potential npm version mismatch. My output looks like this for reference:

```
> npm -v
7.3.0
> node -v
v15.5.0
```